tic_tac_toe/game.py

```diff
@@ -63,22 +71,40 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    for x in range(3):
+        if _is_winning_combination(board, [(x,0), (x,1), (x,2)], player):
+            return player
+        if _is_winning_combination(board, [(0,x), (1,x), (2,x)], player):
+            return player
+    if _is_winning_combination(board, [(0,0), (1,1), (2,2)], player):
```

Comment:
For the diagonals, you might've wanted to just store a `diagonals = [[(0,0), (1,1), (2,2)],[(2,0), (1,1), (0,2)]]` and looped it through.

In fact, you might've been able to do a global one where there's a list of every winning condition, and did `for` loop in this function.